python/fastui/components/forms.py

```diff
@@ -34,6 +34,7 @@ class FormFieldInput(BaseFormField):
 class FormFieldCheckbox(BaseFormField):
     initial: bool | None = None
     type: typing.Literal['FormFieldCheckbox'] = 'FormFieldCheckbox'
+    mode: typing.Literal['checkbox', 'switch'] = 'checkbox'
```

Comment:
`type` should always come last as you'll never want to change it, also I think we should rename the model to `FormFieldBoolean`